src/views/v2/components/icons/cal-export.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * View: Calendar Export Icon
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/v2/components/icons/cal-export.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link http://evnt.is/1aiy
+ *
+ * @since TBD
+ *
+ * @var array<string> $classes Additional classes to add to the svg icon.
+ *
+ * @version TBD
+ *
+ */
+$svg_classes = [ 'tribe-common-c-svgicon', 'tribe-common-c-svgicon--cal-export' ];
+
+if ( ! empty( $classes ) ) {
+	$svg_classes = array_merge( $svg_classes, $classes );
+}
+?>
+<svg <?php tribe_classes( $svg_classes ); ?> viewBox="0 0 23 17" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Is `fill="none"` necessary? I am concerned it will override fill from our stylesheets. 

I'm thinking we can leave it off completely. We do on most of our existing icons.

Reply:
Removed it.